specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json

```diff
@@ -774,6 +771,28 @@
           "x-ms-identifiers": [
             "name"
           ]
+        },
+        "stickySessions": {
+          "description": "Sticky Sessions for Single Revision Mode",
+          "type": "object",
+          "properties": {
+            "affinity": {
+              "description": "Sticky Session Affinity",
+              "type": "string"
+            },
+            "httpOnly": {
+              "description": "Sticky Session HttpOnly",
+              "type": "boolean"
+            },
+            "secure": {
+              "description": "Sticky Session secure",
+              "type": "boolean"
+            },
+            "sameSite": {
+              "description": "Sticky Session sameSite",
+              "type": "string"
+            }
```

Comment:
These are not configurable, why add them to the ARM interface?

Reply:
will remove and leave `affinity` only